test/src/tests.js

```diff
@@ -189,6 +189,121 @@ describe('Rokt Forwarder', () => {
+            window.mParticle.forwarder.launcher = {
+                hashAttributes: function (attributes) {
+                    window.mParticle.Rokt.hashAttributesOptions = attributes;
+                    window.mParticle.Rokt.hashAttributesCalled = true;
+
+                    // Mocking the hashAttributes method to show that
+                    // the attributes will be transformed by the launcher's
+                    // hashAttributes method.
+                    return Promise.resolve({
+                        'test-attribute': 'hashed-value',
+                    });
+                },
+            };
+        });
+
+        it('should call launcher.hashAttributes with passed through attributes when fully initialized', function () {
+            // Ensure both initialization conditions are met
+            window.mParticle.forwarder.isInitialized = true;
+            window.mParticle.forwarder.launcher = {
+                hashAttributes: function (attributes) {
+                    window.mParticle.Rokt.hashAttributesOptions = attributes;
+                    window.mParticle.Rokt.hashAttributesCalled = true;
+                    return {
+                        'test-attribute': 'hashed-value',
+                    };
+                },
+            };
+
+            var attributes = {
+                'test-attribute': 'test-value',
+            };
+
+            window.mParticle.forwarder.hashAttributes(attributes);
+
+            window.Rokt.hashAttributesCalled.should.equal(true);
+            window.Rokt.hashAttributesOptions.should.deepEqual(attributes);
+        });
+
+        it('should return null when launcher exists but kit is not initialized', function () {
+            // Set launcher but ensure isInitialized is false
+            window.mParticle.forwarder.isInitialized = false;
+            window.mParticle.forwarder.launcher = {
+                hashAttributes: function () {},
+            };
+
+            var result = window.mParticle.forwarder.hashAttributes({
+                'test-attribute': 'test-value',
+            });
+
+            (result === null).should.equal(true);
+        });
+
+        it('should log an error when called before initialization', function () {
+            var errorLogged = false;
+            var errorMessage = null;
+            window.console.error = function (message) {
+                errorLogged = true;
+                errorMessage = message;
+            };
+
+            // Ensure kit is not initialized
+            window.mParticle.forwarder.isInitialized = false;
+            window.mParticle.forwarder.launcher = null;
+
+            window.mParticle.forwarder.hashAttributes({
+                'test-attribute': 'test-value',
+            });
+
+            errorLogged.should.equal(true);
+            errorMessage.should.equal('Rokt Kit: Not initialized');
+        });
+
+        it('should return null when kit is initialized but launcher is missing', function () {
```

Comment:
similar to the above, we should expect an error `Rokt Kit: Not initialized`.  Can you add that test?